src/doc/unstable-book/src/library-features/asm.md

```diff
@@ -708,6 +712,12 @@ Some registers have multiple names. These are all treated by the compiler as ide
 | Hexagon | `r30` | `fr` |
 | Hexagon | `r31` | `lr` |
 | BPF | `r[0-10]` | `w[0-10]` |
+| AVR | `xh` | `r27` |
+| AVR | `xl` | `r26` |
+| AVR | `yh` | `r29` |
+| AVR | `yl` | `r28` |
+| AVR | `zh` | `r31` |
+| AVR | `zl` | `r30` |
```

Comment:
These are uppercase in the code.

Reply:
Will address once the capitalization question is answered.